src/components/Popover/Popover.js

```diff
@@ -122,7 +122,10 @@ class Popover extends Component {
         >
           {({ ref, style }) =>
             isOpen && (
-              <div style={style} ref={this.receivePopoverRef}>
+              <div
+                style={{ ...style, zIndex: '10' }}
```

Comment:
Meh, I'm not too happy about the style attribute. Any way of making this a styled component?

Reply:
you can also use the `css` fn is really needed.

Reply:
Styled component for the `zIndex` - yes, but the style in this case is dynamic in nature and can be updated too frequently in some situations, which can have performance implications.

Reply:
@connor-baer